### ISSUE TITLE
Have to dig to find Windows installation method

### ISSUE THREAD

**Issue by vky** [rust-lang/beta.rust-lang.org#423]
The `Installing Rust` section of http://beta.rust-lang.org/learn/get-started has a clear method for  unix-like OS's, but to find an installer for Windows, I have to scroll to the bottom of that page, open up `LEARN MORE` under `Other Installation methods`, scroll down to `Other ways to install rustup` and then I finally find a link to an .exe that may be usable.

With Windows being a major platform, it seems more appropriate (to me) to have an installation method present upfront in the `Installing Rust` section.

**Comment by nagisa** [comment]
It seems that the user-agent detection might not be working right.

**Comment by vky** [comment]
> It seems that the user-agent detection might not be working right.

Is that the case? If it is, then my issue may be a non-issue as I was accessing the page from a Mac.

**Comment by ashleygwilliams** [comment]
we can solve this by making it easy to access the full instructions (something i do on other projects). i am gonna change the title of this to reflect that and then can make a PR :) thanks for filing!

**Comment by vky** [comment]
Tested from a Windows machine, and the Windows installation was indeed present.

I think there's value in having both installations present rather than being conditionally present, but I'll leave that for you all to decide.

**Comment by nicoburns** [comment]
> I think there's value in having both installations present rather than being conditionally present, but I'll leave that for you all to decide.

Agreed. A use case that comes up for me is using my mac to look up how to help a friend using windows to install something. I haven't had to do this for Rust (yet!), but I was doing this only last week for python.

**Comment by ashleygwilliams** [comment]
in case it is not clear, i will be fixing this so that you can see all the options by clicking a button :)

**Comment by SimonSapin** [comment]
> we can solve this by making it easy to access the full instructions

We should do thing regardless of detection improvements. One could be using a web browser on a given machine while working on a different (possibly remote) machine that could be on a different platform. (The case that happened to me was adding Windows servers for Servo CI, from my Linux laptop.)

**Comment by mitsuhiko** [comment]
Ran into this as well trying to look up windows instructions for my windows vm from my host mac.
